gravitee-apim-console-webui/src/management/api/general-info/api-general-info-migrate-to-v4-dialog/api-general-info-migrate-to-v4-dialog.component.scss

```diff
@@ -0,0 +1,122 @@
+@use 'sass:map';
+@use '@angular/material' as mat;
+@use '@gravitee/ui-particles-angular' as gio;
+
+$typography: map.get(gio.$mat-theme, typography);
+
+:host {
+  .title {
+    text-align: left;
+    margin-bottom: 16px;
+  }
+
+  .actions {
+    padding: 8px 24px;
```

Comment:
Same here, I think it's better to keep the values of the application. 
I imagine that Figma is different. But the uniqueness of the application should always take precedence over the creativity of Figma 🤷‍♂️